sdk/tests/deqp/modules/shared/glsBuiltinPrecisionTests.js

```diff
@@ -1117,7 +1117,9 @@ var setParentClass = function(child, parent) {
     glsBuiltinPrecisionTests.ApplyVar.prototype.doEvaluate = function(ctx) {
         return this.m_func.applyFunction(ctx,
                     ctx.env.lookup(this.m_args.a), ctx.env.lookup(this.m_args.b),
```

Comment:
The lookup() function will return a reference/address of an object in native implementation. However, in the js implementation, it returns the value of the object. So the object itself is lost during argument passing in js implementation. 
See the implementation of deEvaluate() in native code https://android.googlesource.com/platform/external/deqp/+/deqp-dev/modules/glshared/glsBuiltinPrecisionTests.cpp#1404. And it can assign value to this referenced object returned by lookup(), see https://android.googlesource.com/platform/external/deqp/+/deqp-dev/modules/glshared/glsBuiltinPrecisionTests.cpp#4556. But in JS implementation, it is invalid to do so. 

As a result, the assignment on line 4571 in this js file can not change the value stored in the original object (ctx.env.m_map). It is useless.